### PR TITLE
bug fix: FlushDB command is not recorded by AOF

### PR DESCRIPTION
**Pull request by suger-no**

### CODE REVIEW

**Comment by suger-no** [comment]
修复重写aof时，老aof文件未被替换

File: database/server.go [hunk 1]
```diff
@@ -197,12 +197,15 @@ func execSelect(c redis.Connection, mdb *Server, args [][]byte) redis.Reply {
 	return protocol.MakeOkReply()
 }
 
-func (server *Server) flushDB(dbIndex int) redis.Reply {
+func (server *Server) flushDB(dbIndex int, isFlushAll bool) redis.Reply {
```
Comment (HDT3213) [inline database/server.go:200]: The parameter `isFulshAll` is very confusing. We could create two functions，a internal function to flush db only invoked by flushAll, and another function invoked by Exec which has AOF and all necessary procedures.

**Review by HDT3213 — CHANGES_REQUESTED** [review]

File: database/server.go (hunk 1 above)
Comment (suger-no) [reply database/server.go]: I was wrong. I thought about it this way. Before that, I thought adding another function was a bad behavior.

**Comment by HDT3213** [comment]
奇怪，怎么 CI 过不去

**Comment by suger-no** [comment]
testaof   这个测试的时候出了问题，找不到key

**Comment by suger-no** [comment]
> 奇怪，怎么 CI 过不去

testaof 这个测试的时候出了问题，找不到key

**Comment by suger-no** [comment]
> testaof 这个测试的时候出了问题，找不到key



> 奇怪，怎么 CI 过不去

现在应该可以了

**Pull request merged**